---
description: How to check and copy-edit the data availability section
---

# Data availability

## What is the data availability section?

The data availability section is intended to provide readers with information on how they can access the data being reported in an article. eLife strongly encourages authors to deposit their generated data in a dedicated, open repository so that it is available for further study and efforts to replicate their results. The data section should also give details of any previously published datasets that the authors used.

![](../../.gitbook/assets/screen-shot-2020-04-27-at-11.15.49.png)

This section consists of three parts - the data availability statement, generated datasets and previously published datasets.

* **Data availability statement**: a short paragraph describing the data that has been generated and where it has been deposited. If all the data is contained in the article and supporting files, or no data was generated at all, this will be stated instead.
* **Generated datasets**: reference details for all generated datasets. These are entered as data references but are listed here rather than in the main reference list. Full bibliographic details are required for each entry \(author list, year, title, database, a direct link and an identifier\).
* **Previously published datasets**: reference details for all previous datasets that were used. Same as the above, only for data that was used in the work reported, rather than generated by it.

In research content, the data availability statement is mandatory. The other two parts are optional, depending on whether any major datasets were generated or used. If only a small amount of data was generated, the authors may chose to include it as source data files for their article rather than submitting it to a repository, meaning no datasets need to be listed in this section. Relatedly, if previously published data was taken wholly from a journal article rather than from a repository, citing that article in the main text is sufficient and no entry in the data availability section is required. Finally, in the case where no data was generated at all, the data availability statement should reflect this.

## What needs to be checked?

### Data availability statement

The data availability statement should be a single paragraph consisting of complete sentences, ending in a full stop. Spelling and grammar errors should be corrected in the same manner as the main text.

The contents of the statement should be reflected in the rest of the dataset section. If specific generated or previously published datasets are mentioned in the statement, these should be listed under the appropriate headings below. If the authors mention using an entire database, however, it will not be appropriate to add a citation entry.

Any statement that indicates data is available on request from the authors will need to be approved by Editorial. Cases where this is acceptable include exceptionally large datasets that cannot be supported by exisitng repositories and patient data that has not been anonymised. Usually in acceptable cases, there will be a note in the 'Production Comments' \(red sticky notes in the submission system\) to indicate that the statement has been OKed by the editors. Alternatively, a note may have been left to indicate that the Editorial team is in contact with the authors to ask them to deposit their data in an open repository. In the absence of any notes, the Production team will need to contact Editorial to clarify what needs to be done.

Data availability statements may also mention code that has been made publicly available by the authors. Instructions on how to deal with this can be found [here](../../toolkit/github/forking-git-based-repos.md#checklist-for-forking-repositories).

### Generated datasets

Each entry under the generated datasets heading should be a complete data citation in eLife style. Individual authors should be listed in 'Surname Intials' style \(e.g. Smith JS\), and a year, title, database and identifier should all be present. A direct URL should also be present for each dataset. Links that just go to the the landing page for a dataset will need to be replaced. See below for examples of how to do this.

In this section only, references that would result in identical citation details \(e.g. two or more datasets that would be Smith et al., 2020\) do not need to be distinguished with a, b, c etc after the year.

Each dataset listed must be publicly available at the point of publication. This can be checked by accessing the provided direct URL. See [below](data-availability.md#data-that-is-not-yet-public) for actions in cases where data is not yet available.

Under no circumstances should journal articles be listed as generated datasets - these should be given in the main reference list only.

### Previously published datasets

The same rules apply to as for generated datasets. 

## Examples

### Adding datasets

Datasets will usually be entered by the authors on the submission system when they are uploading their article. The details will then be included in the XML for content processing. At any point during proofing, datasets may be added or removed. Clicking the 'add dataset' control will bring up the following editor:

![](../../.gitbook/assets/screen-shot-2020-04-27-at-12.53.23.png)

Each of these fields will need to be completed for the dataset to pass schematron validation.

* **Authors**: Enter each author name as a surname followed by initials. A comma separated list \(Smith J, Bloggs T, Muggins I . . . \) may be pasted into this field and will process into separate names when the return key is pressed.
* **Year**: This should be the year that the dataset was made publicly available. For generated datasets, be aware that this made not be the same as the year of publication for the current article.
* **Dataset title**: The title for the dataset. This is often descriptive of the contents, but may also reflect the article to which the data is related \(e.g. 'Data from: Computational 3D histological phenotyping of whole zebrafish by X-ray histotomography' for a Dryad dataset\).
* **Dataset url**: A direct link to the dataset. This should not be a link to a database landing page \(e.g. https://www.rcsb.org/\) but should instead take the reader straight to the data. 
* **Database**: The name of the repository in which the data has been deposited.
* **Identifier**: A unique string that can be used to indentify the dataset. This may be a DOI, an accession number or simply an ID unique to the database in which the data has been stored.
* **pub-id-type**: The type of identifier present. Chose from: accession, archive, doi. This cannot be left as 'other'. Guidance on which databases take which value for this field can be found [here](data-availability.md#database-examples).
* assigning-authority: The organisation that assigned the identifier. Often the organisation that hosts/maintains the database. This can be set to 'other' if multiple organisations are involved or the authority is not on the list of allowed values. The values for common databases can be found [here](data-availability.md#database-examples).

Once all the fields are completed, you dataset can be saved and it will appear in the relevant section. The editor is the same for generated and previously published datasets.

### Incomplete datasets

Sometimes, the authors will not have provided the complete dataset information when they submit the article. This can be because they did not have the finalised details, or simply because they did not want to fill out all the fields at the time. The full details must be added prior to publication.

For example if the authors have simply added 'Smith et al.' or similar in the submission system, the full author list will need to be located and entered. For some datasets, this can be as simple as following the URL, since the page will often show the required details. See [https://www.rcsb.org/structure/4lk1](https://www.rcsb.org/structure/4lk1):

![](../../.gitbook/assets/screen-shot-2020-04-27-at-14.32.57.png)

The same method can be used to identify missing titles, publication dates and so on.

However, some databases do not display the required information in an easily accessible manner. The NCBI BioProject, for example, will usually only show the institution or group from which the data was deposited, rather than the individual researchers responsible.

![](../../.gitbook/assets/screen-shot-2020-04-27-at-14.37.01.png)

In cases where the dataset information cannot be descerned from the link, or the authors have not provided a proper URL in the first place, the following queries should be used to ask the author to provide the missing details. Please combine these where necessary.

* Please provide the full author list for this dataset.
* Please provide the year for this dataset.
* Please provide the accession number for this dataset.

In cases where the authors have not provided a URL, or where they have provided a link to the database homepage rather than a specific dataset, it may be possible to derrive the specific URL. For example, if the dataset is in the NCBI Gene Expression Omnibus, [https://www.ncbi.nlm.nih.gov/geo/](https://www.ncbi.nlm.nih.gov/geo/), and the authors provide an accession number, GSE143881, then the direct URL will be [https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143881](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143881). Further examples of URLs can be found [below](data-availability.md#database-examples). It will usually possible to replace the identifier within the example link with the one the authors have provided to get the final URL. Please note, however, that this link may not be active yet \(this might be why the authors did not provide it\), so action will need to be [taken accordingly](data-availability.md#data-that-is-not-yet-public).

If it is not possible to derrive the URL from the information provided \(e.g. the authors did not provide an identifier, or the database does not have this neatly in individual URLS\), please use the following author query:

* Please enter a direct URL for this dataset, not just the homepage for the database as a whole.

### Data available 'on request'

It is standard practice for a lot of journals to allow authors to make their data available only 'on request', meaning that a reader must contact the corresponding author \(or other designated contact\) to access the data underlying an article. To ensure openness, eLife instead encourages authors to deposit all their data in a dedicated repository or a more general archive such as Dryad or figshare, where it will be available to anyone who wants it.

If an article is submitted with a data availability statement that says generated data is available on request, the Editorial team will ask the authors to upload it to an appropriate repository. Usually there will be no problem with this and the data statement will be updated accordingly. On occasion, the authors may push back against this. This can be because of unfamiliarity with the available options and in these cases, the Editorial team will continue to work with the authors to find an appropriate solution. However, it can also be because the dataset is simply too large \(in the range of terra-bytes of data\) for any repository to accept, necessitating a custom solution such as a dedicated institutional server.

> Example of DAS with large dataset

Another reason for not making the data publicly available would be if it contained identifying information about patients or participants in human trials.

> Example of DAS with private data

In both these cases, 'on request' data is permitted with the approval of the editors handling the article. 



### Data available 'on acceptance'

Related to the above, sometimes when authors are submitting their article, they will hold back data in order to keep it confidential until the article has passsed peer review

> All data generated or analysed during this study are included in the manuscript and supporting files. Source data files will be made available upon acceptance

In these cases, the article should be updated with the full dataset information and/or additional source data files. Idealy this will happen before export from the submission system \(e.g. when the post-peer review revised submission is uploaded\). However, sometimes there will be a delay and an article will move on in the process before the data has been finalised. 

### Data mentioned in the statement but not listed as a citation



### Dataset entries that will need to be removed



### Data that is not yet public

"All data generated or analysed during this study are included in the manuscript and supporting files. Source data files will be made available upon acceptance."





### Database examples

| Database | Synonyms | pub-id-type | assigning-authority | URL Examples | DOI Examples |
| :--- | :--- | :--- | :--- | :--- | :--- |
| Apollo - University of Cambridge Repository | Apollo, University of Cambridge Repository | doi | University of Cambridge | [https://www.repository.cam.ac.uk/handle/1810/260862](https://www.repository.cam.ac.uk/handle/1810/260862) | [https://doi.org/10.17863/CAM.6018](https://doi.org/10.17863/CAM.6018) |
| ArrayExpress | EBI ArrayExpress | accession | EBI | [https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3557/](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3557/) |  |
| CRCNS.org | CRCNS, Collaborative Research in Computational Neuroscience, Collaborative Research in Computational Neuroscience - Data sharing | doi | CRCNS | [http://crcns.org/data-sets/bst/pag-1](http://crcns.org/data-sets/bst/pag-1) | [http://doi.org/10.6080/K0R49P0V](http://doi.org/10.6080/K0R49P0V) |
| Dryad Digital Repository | Dryad | doi | Dryad | [https://datadryad.org//resource/doi:10.5061/dryad.8153g](https://datadryad.org//resource/doi:10.5061/dryad.8153g) | [https://doi.org/10.5061/dryad.8153g](https://doi.org/10.5061/dryad.8153g) |
| EBI European Nucleotide Archive | European Nucleotide Archive, ENA, EBI ENA | accession | EBI | [https://www.ebi.ac.uk/ena/data/search?query=PRJEB18563](https://www.ebi.ac.uk/ena/data/search?query=PRJEB18563) |  |
| Edinburgh DataShare |  | doi | Edinburgh University | [https://datashare.is.ed.ac.uk/handle/10283/1922](https://datashare.is.ed.ac.uk/handle/10283/1922) |  |
| EMDataBank | EMDB, Electron Microscopy Data Bank, The Electron Microscopy Data Bank | accession | EMDB | [http://emsearch.rutgers.edu/atlas/5778\_summary.html](http://emsearch.rutgers.edu/atlas/5778_summary.html) |  |
| EMDataBank | EMDB, Electron Microscopy Data Bank, The Electron Microscopy Data Bank | accession | EMDB | [http://www.ebi.ac.uk/pdbe/entry/emdb/EMD-5778](http://www.ebi.ac.uk/pdbe/entry/emdb/EMD-5778) |  |
| ETH Library Research Collection | ETH Research Collection | doi | ETH Zurich | [https://www.research-collection.ethz.ch/handle/20.500.11850/300799](https://www.research-collection.ethz.ch/handle/20.500.11850/300799) | [https://doi.org/10.3929/ethz-b-000300799](https://doi.org/10.3929/ethz-b-000300799) |
| figshare | Figshare, FigShare | doi | figshare | [https://figshare.com/articles/fMRI\_to\_intraneural\_microstimulation\_of\_single\_mechanoreceptive\_afferents/3205552](https://figshare.com/articles/fMRI_to_intraneural_microstimulation_of_single_mechanoreceptive_afferents/3205552) | [https://dx.doi.org/10.6084/m9.figshare.3205552](https://dx.doi.org/10.6084/m9.figshare.3205552) |
| GeneNetwork | WebQTL | accession | University of Tennessee | [http://www.genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN\_AccessionId=327](http://www.genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=327) |  |
| NCBI BioProject | BioProject | accession | NCBI | [https://www.ncbi.nlm.nih.gov/bioproject/263038](https://www.ncbi.nlm.nih.gov/bioproject/263038) |  |
| NCBI BioSample | BioSample | accession | NCBI | [https://www.ncbi.nlm.nih.gov/biosample/SAMN10501893](https://www.ncbi.nlm.nih.gov/biosample/SAMN10501893) |  |
| NCBI dbGaP | dbGaP | accession | NCBI | [https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study\_id=phs001660.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001660.v1.p1) |  |
| NCBI Gene Expression Omnibus | Gene Expression Omnibus, GEO, NCBI GEO | accession | NCBI | [https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69545](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69545) |  |
| NCBI Nucleotide | Nucleotide | accession | NCBI | [https://www.ncbi.nlm.nih.gov/nuccore/KY616976](https://www.ncbi.nlm.nih.gov/nuccore/KY616976) |  |
| NCBI Sequence Read Archive | Sequence Read Archive, SRA, Short Read Archive, NCBI SRA | accession | NCBI | [https://www.ncbi.nlm.nih.gov/sra/?term=SRA012474](https://www.ncbi.nlm.nih.gov/sra/?term=SRA012474) |  |
| NIMH Data Archive  | NDA | archive | NIMH | [https://nda.nih.gov/edit\_collection.html?id=2890](https://nda.nih.gov/edit_collection.html?id=2890) |  |
| PRIDE | EBI PRIDE, PRoteomics IDEntifications Database | accession | EBI | [https://www.ebi.ac.uk/pride/archive/projects/PXD007787](https://www.ebi.ac.uk/pride/archive/projects/PXD007787) |  |
| ProteomeXchange |  | accession |  | [http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD005831](http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD005831) |  |
| RCSB Protein Data Bank | Protein Data Bank, PDB, RCSB PDB, Protein Databank | accession | PDB | [https://www.rcsb.org/structure/5HOO](https://www.rcsb.org/structure/5HOO) | [http://dx.doi.org/10.2210/pdb5HOO/pdb](http://dx.doi.org/10.2210/pdb5HOO/pdb) |
| Open Science Framework | OSF | archive | Open Science Framework | [https://osf.io/6hmrz/](https://osf.io/6hmrz/) |  |
| Oxford University Research Archive | ORA | archive | Oxford University | [https://ora.ox.ac.uk/objects/uuid:09bef38c-999f-4fb7-aa46-14eda3123571](https://ora.ox.ac.uk/objects/uuid:09bef38c-999f-4fb7-aa46-14eda3123571) |  |
| PeptideAtlas |  | archive | PeptideAtlas | [https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/PASS\_View?identifier=PASS01443](https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/PASS_View?identifier=PASS01443) |  |
| webKnossos |  | archive | webKnossos | [https://wklink.org/8732](https://wklink.org/8732) |  |
| Zenodo |  | doi | Zenodo | [https://zenodo.org/record/6960\#.WqEkGoL7lTY](https://zenodo.org/record/6960#.WqEkGoL7lTY) | [http://dx.doi.org/10.5281/zenodo.6960](http://dx.doi.org/10.5281/zenodo.6960) |

## Schematron warnings

### Content warnings

#### ext-link-child-test-5

**Warning**: _ext-link looks like it points to a review dryad dataset - XXXXXX. Should it be updated?_

**Action**:

#### sec-test-3

**Warning**: _Section has a title 'XXXXXX'. Is it a duplicate of the data availability section \(and therefore should be removed\)?_

**Action**:

#### pre-das-elem-person-group-1

**Warning**: _The reference in position XXXXXX of the data availability section does not have any authors \(no person-group\[@person-group-type='author'\]\). Please ensure to add them in or query the authors asking for the author list._

**Action**: 

#### final-das-elem-person-group-1

**Error**: _The reference in position XXXXXX of the data availability section does not have any authors \(no person-group\[@person-group-type='author'\]\). Please ensure to add them._

**Action**: 

#### das-elem-person-group-2

**Error**: _The reference in position XXXXXX of the data availability has XXXXXX person-group elements, which is incorrect_

**Action**: 

#### pre-das-elem-person-1

**Warning**: _The reference in position XXXXXX of the data availability section does not have any authors. Please ensure to add them in or query the authors asking for the author list._

**Action**: 

#### final-das-elem-person-1

**Error**: _The reference in position XXXXXX of the data availability section does not have any authors \(person-group\[@person-group-type='author'\]\). Please ensure to add them in._

**Action**: 

#### pre-das-elem-data-title-1

**Warning**: _The reference in position XXXXXX of the data availability section does not have a title \(no data-title\). Please ensure to add it in or query the authors asking for it._

**Action**: 

#### final-das-elem-data-title-1

**Error**: _The reference in position XXXXXX of the data availability section does not have a title \(no data-title\). Please ensure to add it_

**Action**: 

#### pre-das-elem-source-1

**Warning**: _The reference in position XXXXXX of the data availability section does not have a database name \(no source\). Please ensure to add it in or query the authors asking for it._

**Action**: 

#### final-das-elem-source-1

**Error**: _The reference in position XXXXXX of the data availability section does not have a database name \(no source\). Please ensure to add it in._

**Action**: 

#### pre-das-elem-pub-id-1

**Warning**: _The reference in position XXXXXX of the data availability section does not have an identifier \(no pub-id\). Please ensure to add it in or query the authors asking for it._

**Action**: 

#### final-das-elem-pub-id-1

**Error**: _The reference in position XXXXXX of the data availability section does not have an identifier \(no pub-id\). Please ensure to add it in._

**Action**: 

#### pre-das-elem-pub-id-2

**Warning**: _The reference in position XXXXXX of the data availability section does not have an id \(pub-id is empty\). Please ensure to add it in or query the authors asking for it._

**Action**: 

#### final-das-elem-pub-id-2

**Error**: _The reference in position XXXXXX of the data availability section does not have an id \(pub-id is empty\). Please ensure to add it in._

**Action**: 

#### pre-das-elem-year-1

**Warning**: _The reference in position XXXXXX of the data availability section does not have a year. Please ensure to add it in or query the authors asking for it._

**Action**: 

#### final-das-elem-year-1

**Error**: _The reference in position XXXXXX of the data availability section does not have a year. Please ensure to add it in._

**Action**: 

#### das-elem-cit-1

**Error**: _Every reference in the data availability section must have an @specific-use. The reference in position XXXXXX does not._

**Action**: 

#### das-elem-cit-2

**Error**: _The reference in position XXXXXX of the data availability section has a @specific-use value of XXXXXX, which is not allowed. It must be 'isSupplementedBy' or 'references'._

**Action**: 

#### das-elem-cit-3

**Error**: _The reference in position XXXXXX of the data availability section has a link \(XXXXXX\) which is the same as another dataset reference in that section. Dataset reference links should be distinct._

**Action**: 

#### das-elem-cit-4

**Warning**: _The reference in position XXXXXX of the data availability section has a pub-id \(XXXXXX\) which is the same as another dataset reference in that section. This is very likely incorrect. Dataset reference pub-id should be distinct._

**Action**: 

#### das-pub-id-1

**Error**: _Each pub-id element must have an @pub-id-type with one of these types: accession, archive, or doi._

**Action**: 

#### das-pub-id-2

**Error**: _Each pub-id element which is not a doi must have an @xlink-href \(which is not empty\)._

**Action**: 

#### das-elem-citation-child-1

**Error**: _Reference in the data availability section has a XXXXXX element in a XXXXXX element which is not allowed._

**Action**: 

#### das-elem-citation-year-1

**Error**: _Every year in a reference must have an @iso-8601-date attribute equal to the numbers in the year. Reference with id XXXXXX has a year 'XXXXXX' but an @iso-8601-date 'XXXXXX'._

**Action**: 

#### pre-das-elem-citation-year-2

**Warning**: _Reference with id XXXXXX has an empty year. Please ensure to add it in or query the authors asking for it._

**Action**: 

#### final-das-elem-citation-year-2

**Error**: _Reference with id XXXXXX has an empty year. Please ensure to add it in._

**Action**: 

#### data-ref-given-names-test-1

**Warning**: _Given names should always be initialised. Ref contains a given names with a string longer than 4 characters - 'XXXXXX' in XXXXXX. Is this a surname captured as given names? Or a fully spelt out given names?_

**Action**: 

#### data-geo-test

Warning: Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/geo' type link, but the database name is not 'NCBI Gene Expression Omnibus' - XXXXXX. Is that correct?

**Action**: 

#### data-nucleotide-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/nuccore' type link, but the database name is not 'NCBI Nucleotide' or 'NCBI GenBank' - XXXXXX. Is that correct?_

**Action**: 

#### data-bioproject-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/bioproject' type link, but the database name is not 'NCBI BioProject' - XXXXXX. Is that correct?_

**Action**: 

#### data-dbgap-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/gap' type link, but the database name is not 'NCBI dbGaP' - XXXXXX. Is that correct?_

**Action**: 

#### data-popset-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/popset' type link, but the database name is not 'NCBI PopSet' - XXXXXX. Is that correct?_

**Action**: 

#### data-sra-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/sra' type link, but the database name is not 'NCBI Sequence Read Archive' - XXXXXX. Is that correct?_

**Action**: 

#### data-biosample-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/biosample' type link, but the database name is not 'NCBI BioSample' - XXXXXX. Is that correct?_

**Action**: 

#### data-protein-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/protein' type link, but the database name is not 'NCBI Protein' - XXXXXX. Is that correct?_

**Action**: 

#### data-assembly-test

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://www.ncbi.nlm.nih.gov/assembly' type link, but the database name is not 'NCBI Assembly' - XXXXXX. Is that correct?_

**Action**: 

#### data-ncbi-test-1

**Warning**: _Data reference with an NCBI link 'XXXXXX' is not marked as an accession number, which is likely incorrect._

**Action**: 

#### data-ncbi-test-2

**Warning**: _Data reference with the database source 'XXXXXX' is not marked as an accession number, which is very likely incorrect._

**Action**: 

#### data-ncbi-test-3

**Warning**: _Data reference with an NCBI link 'XXXXXX' is not marked with NCBI as its assigning authority, which must be incorrect._

**Action**: 

#### data-ncbi-test-4

**Warning**: _Data reference with the database source 'XXXXXX' is not marked with NCBI as its assigning authority, which must be incorrect._

**Action**: 

#### data-dryad-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a Dryad type doi XXXXXX, but the database name is not 'Dryad Digital Repository' - XXXXXX._

**Action**: 

#### data-dryad-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.5061/dryad' or '10.7272', which is incorrect._

**Action**: 

#### data-dryad-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Dryad type doi - XXXXXX, but the assigning authority is not Dryad, which must be incorrect._

**Action**: 

#### data-rcsbpbd-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'http://www.rcsb.org' type link, but the database name is not 'RCSB Protein Data Bank' - XXXXXX. Is that correct?_

**Action**: 

data-rcsbpbd-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'http://www.rcsb.org' type link, but is not marked with PDB as its assigning authority, which must be incorrect_

**Action**: 

#### data-rcsbpbd-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a PDB 'http://www.rcsb.org' type link, but is not marked as an accession type link._

**Action**: 

#### data-emdb-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'http://www.ebi.ac.uk/pdbe/emdb' type link, but the database name is not 'Electron Microscopy Data Bank' - XXXXXX. Is that correct?_

**Action**: 

#### data-emdb-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'http://www.ebi.ac.uk/pdbe/emdb' type link, but is not marked with EMDB as its assigning authority, which must be incorrect_

**Action**: 

#### data-emdb-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a EMDB 'http://www.ebi.ac.uk/pdbe/emdb' type link, but is not marked as an accession type link._

**Action**: 

#### data-arrayexpress-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.ebi.ac.uk/arrayexpress' type link, but the database name is not 'ArrayExpress' - XXXXXX. Is that correct?_

**Action**: 

#### data-arrayexpress-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.ebi.ac.uk/arrayexpress' type link, but is not marked with EBI as its assigning authority, which must be incorrect_

**Action**: 

#### data-arrayexpress-test-3

**Warning**: _Data reference with the title 'XXXXXX' has an ArrayExpress 'www.ebi.ac.uk/arrayexpress' type link, but is not marked as an accession type link._

**Action**: 

#### data-pride-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.ebi.ac.uk/pride' type link, but the database name is not 'PRIDE' - XXXXXX. Is that correct?_

**Action**: 

#### data-pride-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.ebi.ac.uk/pride' type link, but is not marked with EBI as its assigning authority, which must be incorrect_

**Action**: 

#### data-pride-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a PRIDE 'www.ebi.ac.uk/pride' type link, but is not marked as an accession type link._

**Action**: 

#### data-zenodo-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.5281/zenodo' but the database name is not 'Zenodo' - XXXXXX._

**Action**: 

#### data-zenodo-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.5281/zenodo', which is incorrect._

**Action**: 

data-zenodo-test-3

Warning: Data reference with the title 'XXXXXX' has a Zenodo type doi - XXXXXX, but the assigning authority is not Zenodo, which must be incorrect.

**Action**: 

#### data-osf-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://osf.io' type link, but the database name is not 'Open Science Framework' - XXXXXX. Is that correct?_

**Action**: 

#### data-osf-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'https://osf.io' type link, but is not marked with Open Science Framework as its assigning authority, which must be incorrect._

**Action**: 

#### data-osf-test-3

**Warning**: _Data reference with the title 'XXXXXX' has an OSF 'https://osf.io' type link, but is not marked as an archive type link._

**Action**: 

#### data-osf-test-4

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.17605/OSF' but the database name is not 'Open Science Framework' - XXXXXX._

**Action**: 

#### data-osf-test-5

**Warning**: _Data reference with the title 'XXXXXX' has a OSF type doi - XXXXXX, but the assigning authority is not Open Science Framework, which must be incorrect._

**Action**: 

#### data-figshare-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.6084/m9.figshare' but the database name is not 'figshare' - XXXXXX._

**Action**: 

#### data-figshare-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.6084/m9.figshare' - is this correct? Figshare sometimes host for other organisations \(example http://doi.org/10.1184/R1/9963566\), so this may be fine._

**Action**: 

#### data-figshare-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a figshare type doi - XXXXXX, but the assigning authority is not figshare, which must be incorrect._

**Action**: 

#### data-proteomexchange-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'http://proteomecentral.proteomexchange.org/' type link, but the database name is not 'ProteomeXchange' - XXXXXX. Is that correct?_

**Action**: 

#### data-proteomexchange-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'http://proteomecentral.proteomexchange.org/' type link, but is not marked with 'other' as its assigning authority, which must be incorrect._

**Action**: 

#### data-proteomexchange-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a ProteomeXchange 'http://proteomecentral.proteomexchange.org/' type link, but is not marked as an accession type link._

**Action**: 

#### data-openneuro-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.18112/openneuro' but the database name is not 'OpenNeuro' - XXXXXX._

**Action**: 

#### data-openneuro-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.18112/openneuro' or 'openneuro.org/datasets' type link, which is incorrect._

**Action**: 

#### data-openneuro-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a OpenNeuro type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-synapse-test-1

**Warning**: Data reference with the title 'XXXXXX' has a doi starting with '10.7303/syn' but the database name is not 'Synapse' - XXXXXX.

**Action**: 

#### data-synapse-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.7303/syn', which is incorrect._

**Action**: 

#### data-synapse-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Synapse type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-bmrb-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.bmrb.wisc.edu/data\_library/summary' type link, but the database name is not 'Biological Magnetic Resonance Data Bank' - XXXXXX. Is that correct?_

**Action**: 

#### data-bmrb-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.bmrb.wisc.edu/data\_library/summary' type link, but is not marked with 'other' as its assigning authority, which must be incorrect_

**Action**: 

#### data-bmrb-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a BMRB 'www.bmrb.wisc.edu/data\_library/summary' type link, but is not marked as an accession type link._

**Action**: 

#### data-morphdbase-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.morphdbase.de' type link, but the database name is not 'Morph D Base' - XXXXXX. Is that correct?_

**Action**: 

#### data-morphdbase-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.morphdbase.de' type link, but is not marked with 'other' as its assigning authority, which must be incorrect_

**Action**: 

#### data-morphdbase-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Morph D Base 'www.morphdbase.de' type link, but is not marked as an accession type link._

**Action**: 

#### data-mendeley-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.17632' but the database name is not 'Mendeley Data' - XXXXXX._

**Action**: 

#### data-mendeley-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.17632', which is incorrect._

**Action**: 

#### data-mendeley-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Mendeley Data type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-edatashare-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.7488' but the database name is not 'Edinburgh DataShare' - XXXXXX._

**Action**: 

#### data-edatashare-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.7488', which is incorrect._

**Action**: 

#### data-edatashare-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Edinburgh DataShare type doi - XXXXXX, but the assigning authority is not 'Edinburgh University', which must be incorrect._

**Action**: 

#### data-eth-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.3929' but the database name is not 'ETH Library research collection' - XXXXXX._

**Action**: 

#### data-eth-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.3929', which is incorrect._

**Action**: 

#### data-eth-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a ETH Library research collection type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-crcns-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.6080' but the database name is not 'Collaborative Research in Computational Neuroscience' - XXXXXX._

**Action**: 

#### data-crcns-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.6080', which is incorrect._

**Action**: 

#### data-crcns-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a CRCNS type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-morphosource-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.17602' but the database name is not 'MorphoSource' - XXXXXX._

**Action**: 

#### data-morphosource-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.17602', which is incorrect._

**Action**: 

#### data-morphosource-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a MorphoSource type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-neurovault-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'neurovault.org/collections' type link, but the database name is not 'NeuroVault' - XXXXXX. Is that correct?_

**Action**: 

#### data-neurovault-test-2

**Warning**: Data reference with the title 'XXXXXX' has a 'neurovault.org/collections' type link, but is not marked with 'other' as its assigning authority, which must be incorrect

**Action**: 

#### data-neurovault-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a NeuroVault 'neurovault.org/collections' type link, but is not marked as an archive type link._

**Action**: 

#### data-wwpdb-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.2210' but the database name is not 'Worldwide Protein Data Bank' - XXXXXX._

**Action**: 

data-wwpdb-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.2210', which is incorrect._

**Action**: 

#### data-wwpdb-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Worldwide Protein Data Bank type doi - XXXXXX, but the assigning authority is not 'PDB', which must be incorrect._

**Action**: 

#### data-sbgdb-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.15785/SBGRID' but the database name is not 'SBGrid Data Bank' - XXXXXX._

**Action**: 

#### data-sbgdb-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.15785/SBGRID', which is likely incorrect._

**Action**: 

#### data-sbgdb-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a SBGrid Data Bank type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-harvard-dataverse-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a doi starting with '10.7910' but the database name is not 'Harvard Dataverse' - XXXXXX._

**Action**: 

#### data-harvard-dataverse-test-2

**Warning**: _Data reference with the title 'XXXXXX' has the database name XXXXXX, but no doi starting with '10.7910', which is likely incorrect._

**Action**: 

#### data-harvard-dataverse-test-3

**Warning**: _Data reference with the title 'XXXXXX' has a Harvard Dataverse type doi - XXXXXX, but the assigning authority is not 'other', which must be incorrect._

**Action**: 

#### data-encode-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.encodeproject.org' type link, but the database name is not 'ENCODE' - XXXXXX. Is that correct?_

**Action**: 

#### data-encode-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.encodeproject.org' type link, but is not marked with 'other' as its assigning authority, which must be incorrect_

**Action**: 

#### data-encode-test-3

**Warning**: _Data reference with the title 'XXXXXX' has an ENCODE 'www.encodeproject.org' type link, but is not marked as an archive type link._

**Action**: 

#### data-emdr-test-1

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.emdataresource.org' type link, but the database name is not 'EMDataResource' - XXXXXX. Is that correct?_

**Action**: 

#### data-emdr-test-2

**Warning**: _Data reference with the title 'XXXXXX' has a 'www.emdataresource.org' type link, but is not marked with 'other' as its assigning authority, which must be incorrect_

**Action**: 

#### data-emdr-test-3

**Warning**: _Data reference with the title 'XXXXXX' has an EMDataResource 'www.emdataresource.org' type link, but is not marked as an accession type link._

**Action**: 

### XML structure warnings

#### back-test-3

**Error**: _One and only one Data availiability section \(sec\[@sec-type="data-availability"\]\) must be present \(as a child of back\) for 'XXXXXX'._

**Action**: 

#### back-test-10

**Warning**: _One and only one Data availiability section \(sec\[@sec-type="data-availability"\]\) should be present \(as a child of back\) for 'XXXXXX'. Is this a new version which was published first without one? If not, then it certainly needs adding._

**Action**: 

#### data-p-presence

**Error**: _At least one p element must be present in sec\[@sec-type='data=availability'\]._

**Action**: 

## XML structure

```markup
<sec sec-type="data-availability" id="s7">
    <title>Data availability</title>
    <p>A data availability statement will generally describe how the authors
     have provided the source data for their work. This can list the source 
     data files accompanying their figures, supplementary files, and/or 
     external dataasets. Hyperlinks can be included here, for example: 
     <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102999">
     https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102999</ext-link>.</p>
     <p>The following dataset was generated:</p>
     <p>
         <element-citation publication-type="data" specific-use="isSupplementedBy" id="dataset1">
            <person-group person-group-type="author">
                 <name><surname>Düsterwald</surname><given-names>KM</given-names></name>
                 <name><surname>Currin</surname><given-names>CB</given-names></name>
                 <name><surname>Burman</surname><given-names>RJ</given-names></name>
                 <name><surname>Akerman</surname><given-names>CJ</given-names></name>
                 <name><surname>Kay</surname><given-names>AR</given-names></name>
                 <name><surname>Raimondo</surname><given-names>JV</given-names></name>
             </person-group>
             <year iso-8601-date="2018">2018</year>
             <data-title>Data from: Biophysical models reveal the relative importance
              of transporter proteins and impermeant anions in chloride homeostasis</data-title>
             <source>Dryad Digital Repository</source>
             <pub-id assigning-authority="Dryad" pub-id-type="doi">10.5061/dryad.kj1f3v4</pub-id>
         </element-citation>
     </p>
     <p>The following previously published datasets were used:</p>
     <p>
         <element-citation publication-type="data" specific-use="references" id="dataset2">
             <person-group person-group-type="author">
                 <name><surname>Rau</surname><given-names>CD</given-names></name>
                 <name><surname>Wang</surname><given-names>J</given-names></name>
                 <name><surname>Wang</surname><given-names>Y</given-names></name>
                 <name><surname>Lusis</surname><given-names>AJ</given-names></name>
             </person-group>
             <year iso-8601-date="2013">2013</year>
             <data-title>Transcriptomes of the hybrid mouse diversity panel subjected 
             to Isoproterenol challenge</data-title>
             <source>NCBI Gene Expression Omnibus</source>
             <pub-id assigning-authority="NCBI" pub-id-type="accession" xlink:href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48760">GSE48760</pub-id>
         </element-citation>
     </p>
     <p>
         <element-citation publication-type="data" specific-use="references" id="dataset3">
             <person-group person-group-type="author">
                 <name><surname>Garcia</surname><given-names>Miguel A</given-names></name>
             </person-group>
             <year iso-8601-date="2018">2018</year>
             <data-title>Shear Manuscript</data-title>
             <source>Open Science Framework</source>
             <pub-id assigning-authority="Open Science Framework" pub-id-type="archive" xlink:href="https://osf.io/kvu5j/">kvu5j</pub-id>
         </element-citation>
     </p>
</sec>
```

